sample-app/src/main/java/com/splunk/android/sample/FirstFragment.java

```diff
@@ -206,7 +205,6 @@ private Call.Factory buildOkHttpClient(SplunkRum splunkRum) {
             SSLSocketFactory sslSocketFactory = sslContext.getSocketFactory();
             return splunkRum.createRumOkHttpCallFactory(
                     builder.sslSocketFactory(sslSocketFactory, (X509TrustManager) trustAllCerts[0])
-                            .hostnameVerifier(new AllowAllHostnameVerifier())
```

Comment:
This was giving a deprecation warning -- and the new online store url has a valid cert, so no longer needed.